packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -478,6 +481,25 @@ class AndroidViewController {
 
   MessageCodec<dynamic> _creationParamsCodec;
 
+  final List<PlatformViewCreatedCallback> _platformViewCreatedCallbacks = <PlatformViewCreatedCallback>[];
+
+  /// Whether the platform view has already been created.
+  bool get isCreated => _state == _AndroidViewState.created;
+
+  /// Adds a callback that will get invoke after the platform view has been
+  /// created.
+  void addOnPlatformViewCreatedListener(PlatformViewCreatedCallback listener) {
+    assert(listener != null);
+    assert(_state != _AndroidViewState.disposed);
+    _platformViewCreatedCallbacks.add(listener);
+  }
+
+  /// Removes a callback added with [addOnPlatformViewCreatedListener].
```

Comment:
It might be a little confusing that the callback that is passed to the constructor can be removed here.

We could remove the onPlatformViewCreated constructor parameter, this will be a breaking change but I highly doubt it will affect anyone at this point.

Alternatively I guess we can assert here that we are not removing the callback that was passed to the constructor? (I guess this will be equivalent to maintaining the listeners set separately from the one callback that is passed to the constructor)

Reply:
Yeah, let's remove the constructor parameter.